docs/sdd.md

```diff
@@ -189,310 +181,553 @@ sequenceDiagram
+                psql_linked->>+psql: get_psql_pool()
+                psql-->>psql_linked: <Pool>
             option No connection
-                psql-->>-grpc: Database pool not initialized
+                psql-->>-psql_linked: Database pool not initialized
             end
         end
-        grpc->>+psql_vp: Query vertiport for ID
-        alt No vertiport found
-            psql_vp-->>grpc: <ArrErr>
-            grpc-->>client: <Status<not found>>
-        else vertiport exists
-            psql_vp-->>-grpc: <VertiportPsql>
-            grpc->>+psql_vp: Update vertiport with data
-            alt Update failed
-                psql_vp-->>grpc: <ArrErr>
-                grpc-->>client: <Status<Internal>>
-            else Update success
-                psql_vp-->>psql_vp: read
-                psql_vp-->>-grpc: <VertiportPsql>
-                grpc-->>grpc: <Vertiport> from <VertiportPsql>
-                grpc->>memdb: insert Vertiport
-                grpc-->>-client: <Vertiport>
+        psql_linked-->>-grpc_service: Result<Vec<Row>, Error>
+        alt Err (database error)
+            grpc_service-->>grpc_service: Err(ArrErr::Error)
+        else Ok (query success)
+            grpc_service-->>-grpc_service: Ok(ids)
+            grpc_service->>+psql_search: advanced_search(AdvancedSearchFilter)
+            rect rgb(247, 161, 161)
+                critical Get DB connection from the pool
+                    psql_search->>+psql: get_psql_pool()
+                    psql-->>psql_search: <Pool>
+                option No connection
+                    psql-->>-psql_search: Database pool not initialized
+                end
+            end
+            psql_search-->>-grpc_service: Result<Rows, Error>
+            alt Err (database error)
+                grpc_service-->>grpc_server: Status(Code::Internal)
+            else Ok (search success)
+                grpc_service-->>grpc_service: <IdList> from <Rows>
+                grpc_service-->>-grpc_server: Ok(tonic::Response<IdList>)
             end
         end
     end
+    grpc_server-->>-client: Result
 ```
 
-#### `delete_vertiport`
+#### `get_linked`
 ```mermaid
 sequenceDiagram
     participant client as grpc_client
-    participant grpc as vertiport_rpc_server
-    participant memdb as memdb
+    participant grpc_server as GrpcServer
+    participant grpc_service as grpc::GrpcLinkService
+    participant psql_simple as postgres::simple_resource::PsqlType
+    participant psql_linked as postgres::linked_resource::PsqlType
+    participant psql_search as postgres::PsqlSearch
     participant psql as postgres
-    participant psql_vp as vertiport/psql
-    client->>+grpc: Delete vertiport with ID
+    client->>+grpc_server: get_linked(Request<Id>)
+    grpc_server->>+grpc_service: generic_get_linked(Request<Id>)
+    grpc_service->>+grpc_service: _get_linked(Id)
+    grpc_service->>+psql_simple: get_by_id(Uuid)
     rect rgb(247, 161, 161)
```

Comment:
This is a bit hard on the eyes in dark mode: 
![image](https://user-images.githubusercontent.com/6334638/229329394-5ef439dc-0331-400e-9fe3-44a21664a293.png)

Reply:
Ouch, I can see why, I'll find better colors

Reply:
fixed: 92bbff7ed73eefb66fe197b28d2f13220dc47888

Using Arrow blue, seems to work ok for both black and white theme